charts/updatetree/README.md

```diff
@@ -0,0 +1,32 @@
+# updatetree
+
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.4.3](https://img.shields.io/badge/AppVersion-0.4.3-informational?style=flat-square)
+
+Update the status of an existing Trillian tree
+
+**Homepage:** <https://sigstore.dev/>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| The Sigstore Authors |  |  |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| args.treeID | string | `nil` |  |
+| args.treeState | string | `nil` |  |
```

Comment:
maybe in the description, spell out what are valid treeState values?

Reply:
good call, done